File: PspDataPresenter.App/data-presenter-app/src/app/_core/models/voting-result.model.ts
import { ClubModel } from "./club.model";
import { VotingSummaryModel } from "./voting-summary.model";

export interface VotingResultModel {
    votingClubResults: VotingClubResultModel[];
    summary: VotingSummaryModel;
}

export interface VotingClubResultModel
{
    clubModel: ClubModel;
    for: number;
    abstain: number;
    absent: number;
}

File: PspDataPresenter.Core/Models/VotingResultModel.cs
using System.Collections.Generic;

namespace PspDataPresenter.Core.Models
{
    public class VotingResultModel
    {
        public List<VotingClubResultModel> VotingClubResults { get; set; }
        public VotingSummaryModel Summary { get; set; }
    }
}
